source/rc.php-fpm/php-fpm-log/env/LOG_SCRIPT

```diff
@@ -0,0 +1,2 @@
+n4 s500000 T $LOGDIR_PATH
```

Comment:
It seems php-fpm already adds timestamp by default, so let's drop the `T` directive.